templates/layouts/main.php

```diff
@@ -4,45 +4,50 @@
 <!--[if IE 8]>         <html class="no-js lt-ie9" <?php language_attributes(); ?>> <![endif]-->
 <!--[if gt IE 8]><!--> <html class="no-js" <?php language_attributes(); ?>> <!--<![endif]-->
 <head>
-    <meta charset="utf-8">
-    <?php wp_head(); ?>
+  <meta charset="utf-8">
+  <title><?php wp_title('|', true, 'right'); ?></title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- Prefetch external asset dns -->
+  <link rel="dns-prefetch" href="#">
+
+  <!-- Prefetch internal image assets -->
+  <link rel="prefetch" href="#">
+
+  <link rel="apple-touch-icon-precomposed" href="<?php bloginfo('template_directory'); ?>/templates/assets/img/apple-touch-icon-precomposed.png">
+
+  <link rel="icon" type="image/png" href="<?php bloginfo('template_directory'); ?>/templates/assets/img/shortcut-icon.png" />
+
+  <?php wp_head(); ?>
+
+  <link rel="alternate" type="application/rss+xml" title="<?php echo get_bloginfo('name'); ?> Feed" href="<?php echo home_url(); ?>/feed/">
```

Comment:
Take a look at the new `main.php` - I've moved most of what was in HEAD into `app/Theme/Scripts.php`.